### ISSUE TITLE
Implement training loop

### ISSUE THREAD

**Issue by TaillessSheep** [TaillessSheep/ARIN-5101-Project#6]
Train baseline model tracking loss/accuracy for train and validation. Save plots/logs to /reports/.

**Source:** Slide 9 – Validation Setup
